src/deepqmc/cli.py

```diff
@@ -246,3 +251,79 @@ def evaluate_at(workdir, cuda, store_steps, hook):
         workdir=workdir,
         **params.get('evaluate_kwargs', {}),
     )
+
+
+def setup(rank, world_size):
+    os.environ['MASTER_ADDR'] = 'localhost'
+    os.environ['MASTER_PORT'] = '12345'
+    dist.init_process_group('nccl', rank=rank, world_size=world_size)
+
+
+def cleanup():
+    dist.destroy_process_group()
+
+
+def _train_at_parallel(rank, workdir, save_every, n_gpus, state, sync_processes):
```

Comment:
is it essentially `train_at()` plus setting the `PARALLEL` flag minus the restart logic?

Reply:
Its essentially `train_at()` with the PARALLEL flag and some additional setup for running the training on multiple gpus. This involes the `setup()` and the `cleanup()` functions as well as `mp.spawn`. For now it doesn't contain the restart logic, but I will add that in eventually. Also, this version of the cli will be deprecated and replaced with the hydrabased cli, so this is meant as a temporary solution and will be updated. I will furthermore add an `evaluate_parallel`  function, that facilitates running an evaluation on multiple gpus of a device.

Reply:
I was just wondering if some of the code could be just straight shared with `train_at()`. But that's overthinking it at this point.